server/utils/utils.ts

```diff
@@ -51,7 +51,7 @@ export const compareByDateAndTime =
   (a: T, b: T): number => {
     const aValue = moment(f(a))
     const bValue = moment(f(b))
-    return aValue.unix() - bValue.unix()
+    return bValue.unix() - aValue.unix()
```

Comment:
rename method to `compareByDescendingDateAndTime` ?

Reply:
ok